test/run_tests.sh

```diff
@@ -3,12 +3,19 @@
 set -o errexit
 set -o nounset
 set -o pipefail
+set -o xtrace
 
 # "TARGET" is a Makefile target that runs tests
 TARGET=$1
 
 INTEGRATION_OUTPUT_JUNIT=${INTEGRATION_OUTPUT_JUNIT:-false}
 
+CONTAINER_POSTFIX=$(< /dev/urandom base64 | tr -dc '[:alpha:]' | head -c 8 || true)
```

Comment:
Hm, do we really want to continue if this is too short (possibly empty)?

Reply:
This is to avoid a `SIGPIPE`.